test/services/setup_creator.js

```diff
@@ -50,6 +50,20 @@ describe('Setup Creator', () => {
     });
   });
 
+  describe('createTosFile', () => {
+    const exampleTosText = '0x1234deadface';
+    const tosFilePath = `${testOutputDir}TOS.txt`;
+
+    afterEach(async () => {
```

Comment:
It makes more sense to put it in `beforeEach`.

Reply:
we want to cleanup after the test. Ideally it should be in both places